doc/whats_new/v0.20.rst

```diff
@@ -67,6 +67,13 @@ Changelog
   those estimators as part of parallel parameter search or cross-validation.
   :issue:`12122` by :user:`Olivier Grisel <ogrisel>`.
 
+:mod:`sklearn.metrics`
+........................
+
+- |Fix| Fixed a bug in :func:`metrics.silhouette_score` where diagonal entries
+  of a precomputed distance matrix affected the final score.
+  :issue:`12258` by `Stephen Tierney`.
```

Comment:
Use :user: here

Reply:
Sorry, I followed the bottom example, which seems to be missing the user tag.

```
- |Fix| Fixed bug in :class:`preprocessing.OrdinalEncoder` when passing
  manually specified categories. :issue:`12365` by `Joris Van den Bossche`_.
```